docs/getting-started.md

```diff
@@ -73,7 +73,7 @@ dockerized environment, clone the KubeVirt repository, `cd` into it, and:
 
 ```bash
 # Build and deploy KubeVirt on Kubernetes in our vms inside containers
-export KUBEVIRT_PROVIDER=k8s-1.18 # this is also the default if no KUBEVIRT_PROVIDER is set
+export KUBEVIRT_PROVIDER=k8s-1.20 # this is also the default if no KUBEVIRT_PROVIDER is set
```

Comment:
@yingbaiibm how about removing this line? we will choose the current default anyway. This way we avoid that this line gets outdated.

Reply:
That's also a choice.   But how about changing the comment to  "Find supported K8s version under KUBEVIRT_TOP/cluster-up/cluster and set required KUBEVIRT_PROVIDER.  If no KUBEVIRT_PROVIDER is set, default value will be used"      In this way, user will be clear about how to set the KUBEVIRT_PROVIDER and choose the version they want.

Reply:
Works for me too. Anything which does not mention an explicit version (and therefore needing a regular bump) is an improvement.